channels/us.m3u

```diff
@@ -5085,8 +5063,6 @@ https://linear-43.frequency.stream/dist/localnow/43/hls/master/playlist.m3u8
 https://linear-45.frequency.stream/dist/roku/45/hls/master/playlist.m3u8
 #EXTINF:-1 tvg-id="Revry.us" tvg-name="Revry" tvg-country="US" tvg-language="English" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/5fc81016d98cab623846a4f3" group-title="Lifestyle",Revry (720p)
 https://revry-revry-3-eu.rakuten.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="Revry.us" tvg-name="Revry" tvg-country="US" tvg-language="English" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/5fc81016d98cab623846a4f3" group-title="Lifestyle",Revry (720p) [Offline]
-https://linear-5.frequency.stream/dist/plex/5/hls/master/playlist.m3u8
```

Comment:
Not 24/7

![image](https://user-images.githubusercontent.com/30985701/132143469-c89433a9-b606-4a5b-afe8-5779857e149e.png)